airbyte-integrations/connectors/source-azure-blob-storage/setup.py

```diff
@@ -5,7 +5,20 @@
 
 from setuptools import find_packages, setup
 
-MAIN_REQUIREMENTS = ["airbyte-cdk>=0.51.17", "smart_open[azure]", "pytz", "fastavro==1.4.11", "pyarrow"]
+MAIN_REQUIREMENTS = [
+    "airbyte-cdk>=0.51.17",
+    "smart_open[azure]",
+    "pytz",
+    "fastavro==1.4.11",
+    "pyarrow",
+    "unstructured==0.10.19",
```

Comment:
This stuff is also included in `airbyte-cdk[file-based]`, but then there would be a version mismatch with fastavro 1.4.11 which is listed explicitly here (it's `fastavro~=1.8.0` in the file-based extra). Is this a leftover or is there a strong reason to not rely on the CDK "standard" dependencies for file based sources?

Reply:
I read this comment after I submitted my first review.

Reply:
According to your PR title I would assume that these two connectors would depend on the file based CDK and the unstructured lib would be bundled in it. It's be great if connectors and CDK could agree on the `fastavro` version indeed.

Reply:
I'm not aware of any reason not to use the file-based CDKs dependencies.

Reply:
Switched it to use the extra